server_config.h

```diff
@@ -0,0 +1,19 @@
+#ifndef SERVER_CONFIG_H
+#define SERVER_CONFIG_H
+
+#include "config_parser.h"
+#include <unordered_map>
+
+class ServerConfig {
+  public:
+  	ServerConfig(NginxConfig config);
+    ~ServerConfig();
+    std::string propertyLookUp(std::string propertyName);
+
+  private:
+  	void printPropertiesMap();
+  	void fillOutMap(NginxConfig config);
+  	std::unordered_map<std::string,std::string> property_to_values_;
+};
+
+#endif
```

Comment:
Can you add a newline at the end of this file? That'll let us conform to POSIX conventions where a file terminates with a newline.